js/script.js

```diff
@@ -0,0 +1,59 @@
+const bookForm = document.getElementById('bookForm');
+const booksContainer = document.querySelector('.books');
+const titleInput = bookForm.title;
+const authorInput = bookForm.author;
+const books = JSON.parse(localStorage.getItem('books')) || [];
+
+const addBook = (title, author) => {
+  books.push({
+    title,
+    author,
+  });
+
+  localStorage.setItem('books', JSON.stringify(books));
+
+  return { title, author };
+};
+
+const createBookElement = ({ title, author }) => {
+  // Create elements
+
+  const bookDiv = document.createElement('div');
+  const bookTitle = document.createElement('h2');
+  const bookAuthor = document.createElement('h3');
+  const bookRemove = document.createElement('button');
+  const bookHr = document.createElement('hr');
+
+  // Fill the content
+  bookTitle.innerText = `Book title: ${title}`;
+  bookAuthor.innerText = `Book author: ${author}`;
+  bookRemove.innerText = 'Remove';
+
+  bookRemove.onclick = function () {
+    bookRemove.parentElement.remove();
+    let myIndex;
+    for (let i = 0; i < books.length; i += 1) {
+      if (books[i].title === title && books[i].author === author) {
+        myIndex = i;
+      }
+    }
+    books.splice(myIndex, 1);
+    localStorage.setItem('books', JSON.stringify(books));
+  };
+
+  // Add to the DOM
+  bookDiv.append(bookTitle, bookAuthor, bookRemove, bookHr);
+  booksContainer.appendChild(bookDiv);
+};
+books.forEach(createBookElement);
+
+bookForm.onsubmit = (e) => {
+  e.preventDefault();
+
+  const newBook = addBook(titleInput.value, authorInput.value);
+
+  createBookElement(newBook);
+
+  titleInput.value = '';
+  authorInput.value = '';
+};
```

Comment:
-  Good job working on this milestone 💪🏿 
- Kindly make sure that when a book  a user clicks the "Add" button:
- [ ]  A new book is added to the collection and the new book is displayed on the page.


- When a user clicks the "Remove" button: 
- [ ] The right book is removed from the collection and  the right book disappears from the page.

https://user-images.githubusercontent.com/69638013/170052530-44970350-9ea7-4530-94d5-c18bdc38537b.mp4